docs/lists.rst

```diff
@@ -0,0 +1,502 @@
+colon, *a stopping index*, then a closing bracket.
+
+.. code-block:: console
+
+    >>> primes
+    [2, 3, 5, 7, 11, 13, 17, 19]
+    >>> primes[2:5]
+    [5, 7, 11]
+
+
+The result is a sublist that starts at index 2, and continues until it reaches
+index 5. Be careful, slicing includes the value at the starting index, but
+excludes the stopping index. The beginning value is included, the ending value is not.
+
+One more slice...
+
+.. code-block:: console
+
+    >>> primes
+    [2, 3, 5, 7, 11, 13, 17, 19]
+    >>> primes[0:6]
+    [2, 3, 5, 7, 11, 13]
+
+
+This will start at the beginning, which is index 0, and continue to index 6,
+which is 17. It will not include the final number, so this slice includes the
+primes from 2 through 13, in other words: the first 5 values.
+
+Notice, that if you start from the beginning, you can ommit the 0 completely
+and the slice will assume that you want to start from index 0. Similarly, if
+you omit the stopping index it will assume that you want to go the end of the
+list.
+
+.. code-block:: console
+
+    >>> primes
+    [2, 3, 5, 7, 11, 13, 17, 19]
+    >>> primes[:6]
+    [2, 3, 5, 7, 11, 13]
+    >>> primes[6:]
+    [17, 19]
+
+
+Multiple data types
+###################
+
+Lists can contain more than prime numbers. They can contain integers, booleans,
+strings, floats, and even other lists.
+
+.. code-block:: python
+
+    examples = [128, True, "Alphabet", 3.14, [32, 64, False]]
+    print(examples)
+
+
+Many languages require lists to contain values of the same type, but not
+Python. With Python you are free to insert multiple data types in the same
+list. Lists can also contain duplicate values. Here is another way lists
+are different from sets. For example, suppose you want to record the
+numbers you roll on a pair of dice. Pretent you roll a 4, 7, 2, 7, 12, 4 and 7.
+
+.. code-block:: console
+
+    >>> rolls = [4, 7, 2, 7, 12, 4, 7]
+    >>> rolls
+    [4, 7, 2, 7, 12, 4, 7]
+
+
+If you look at the list, all the values are there, even the repeated rolls. You
+can also combine lists. To see how, create two separate lists: a list of
+numbers and a list of letters... To combine these two lists into a single list
+use the plus sign.
+
+.. code-block:: console
+
+    >>> numbers = [1, 2, 3]
+    >>> letters = ["a", "b", "c"]
+    >>> numbers + letters
+    [1, 2, 3, 'a', 'b', 'c']
+
+
+But order matters, if you reverse this and compute ``letters + numbers`` you
+get ``'a', 'b', 'c', 1, 2, 3``. Combining lists is called concatenation.
+Observe. The list of numbers and the list of letters are unchanged.
+
+.. code-block:: console
+
+    >>> letters + numbers
+    ['a', 'b', 'c', 1, 2, 3]
+    >>> numbers
+    [1, 2, 3]
+    >>> letters
+    ['a', 'b', 'c']
+
+
+There are many other methods for working with lists. To see them all, pass any list to the directory function. To learn how to use one of these methods, use the help function. For example, there is a method for reversing the list. The
```

Comment:
This part below was not so easy to follow. After reading three times, got it and could see the methods. 
Can't say that help part brings for me (beginner) much value, plus it can be overwhelming seeing all these methods...

Reply:
I replaced with a link to the official python docs.